src/Loader.php

```diff
@@ -99,9 +99,7 @@ private function load_services( $services ) {
 			}
 
 			// Load `CampaignExporter` class on admin campaign listing page and campaign export only.
-			if ( 'campaign' === filter_input( INPUT_GET, 'post_type', FILTER_SANITIZE_STRING ) || 'export_data' === filter_input( INPUT_GET, 'action', FILTER_SANITIZE_STRING ) ) {
-				$this->default_services[] = CampaignExporter::class;
-			}
+			$this->default_services[] = CampaignExporter::class;
```

Comment:
Need to check if it's safe to remove the whole condition.

Reply:
Didn't cause any issues :+1:

Reply:
I think we should keep the second condition, 
`'export_data' === filter_input( INPUT_GET, 'action', FILTER_SANITIZE_STRING )`
It will make sure the class is only loaded when admin export the post data.

Reply:
The `Exporter` class is needed to add buttons to the admin listing view. In that case that condition won't be true. It previously worked on campaign pages because it's an OR and it matched the first part.

Reply:
ok